### ISSUE TITLE
Language in Template not clickable

### ISSUE THREAD

**Issue by justkeen** [khalwat/seomatic#69]
I have English (primary language) and Dutch but whenever I add a template meta, Dutch is not clickable.

**Comment by khalwat** [comment]
Yes, unfortunately this is a limitation in SEOmatic currently (it's listed in the Roadmap section of the docs).

Template Metas are currently not savable localized; I need to spent some time to figure out the right way to do it.

In the interim, you can use the SEOmatic FieldType attached to a Single (which does support localization properly) or you can set whatever you want via Dynamic Twig Meta as described in the wiki docs: https://github.com/nystudio107/seomatic/wiki/07.-Dynamic-Twig-SEO-Meta

**Comment by dowadidi** [comment]
I was able to create templates in other languages by appending the language to the cp url. In yout case "/nl" should enable you to create localized template meta. Not sure if they work if you're using the same template file for both languages though...
